lib/ansible/plugins/action/reboot.py

```diff
@@ -274,8 +283,9 @@ def run(self, tmp=None, task_vars=None):
 
         if reboot_result['failed']:
             result = reboot_result
-            elapsed = datetime.utcnow() - reboot_result['start']
-            result['elapsed'] = elapsed.seconds
+            if 'start' in reboot_result:
```

Comment:
I would like to always return `elapsed`, which means always returning `result['start']`. This is done [here](https://github.com/ansible/ansible/blob/b03feb6d40944fa247dec8adc906e9fa4798522b/lib/ansible/plugins/action/reboot.py#L180-L180) in `reboot.py`. That change didn't make it to `win_reboot.py`. Sorry about that.